### DISCUSSION TITLE
I have an issue where the notification setting on websites aren't permanent, even if the value of 'content.notifications.enabled' is 'true

### DISCUSSION THREAD

**Discussion by groggone** [qutebrowser/qutebrowser#7941] [Q&A]
What i mean is that the notification option being turned on especially in websites such as Discord and Youtube doesn't remain when you close Qutebrowser, you come back and reopen the browser and the notification doesn't even pop up.

On Discord, the notification doesn't pop up while the notification option still appears to be turned on, and on Youtube, the notification will never even pop up because the notification option is turned back off.

**Comment by The-Compiler** [comment]
Might be #5233 that you're seeing.

**Reply by groggone** [reply]
Yes this is the issue i'm seeing
